x/wasm/keeper/metrics.go

```diff
@@ -0,0 +1,65 @@
+package keeper
+
+import (
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	labelPinned = "pinned"
+	labelMemory = "memory"
+	labelFs     = "memory"
```

Comment:
I don't know what those labels are used for, but should this be `"fs"`?

Reply:
🤦  definitively not "memory" thanks for finding this!